### ISSUE TITLE
Bad warning when retrieving plan for connected account

### ISSUE THREAD

**Issue by deatheragetr** [stripe/stripe-node#393]
Stripe-node version: 5.1.0;
Node version: 8.6.0 (Oddly, I only discovered this on upgrading from node 6.x.x to 8.x.x)

Behavior works as expected for retrieving a subscription plan on a connected account; however, a warning is logged.  I believe this warning is wrong.

``` 
stripe
.plans
.retrieve('foo-sub-plan', { stripe_account: 'foo-stripe-account' })
.then(plan => console.log(plan))
```
logs the following:
```
Stripe: Options found in arguments (stripe_account). Did you mean to pass an options object? See https://github.com/stripe/stripe-node/wiki/Passing-Options.

{
  id: 'foo-sub-plan'
  ....plan-details
}
```
Since, I'm calling `plans.retrieve` the way the documentation suggests, and since it's behaving as I would expect, I assume that's a bad warning.

Anyway, confused me a bit as I was debugging a separate, but subscription-plans related issue.

Best,

**Comment by jlomas-stripe** [comment]
That ... is interesting. :)  We recently added that warning, but it definitely shouldn't be appearing there.  I'll have a look at this shortly - thanks for bringing it up!

**Comment by deatheragetr** [comment]
Of course, thanks for the quick response!

**Comment by brandur-stripe** [comment]
Thanks for reporting this! We've released a fix in 5.1.1.

**Comment by deatheragetr** [comment]
Great thanks for the quick turnaround!
